### ISSUE TITLE
Contributing to Ember.js / Adding new features

### ISSUE THREAD

**Issue by danilovaz** [ember-brasil/website#28]
https://github.com/ember-brasil/website/blob/master/source/contributing/adding-new-features.md

- [ ] adding-new-features.md

**Comment by danilovaz** [comment]
Talvez tenha algo aqui que possa ajudar na tradução:
https://github.com/emberjs/guides/blob/e39f8513cfdb2f761af628e7b52f0eafb2335a2d/source/localizable/contributing/adding-new-features.pt-BR.md

**Comment by yanmagale** [comment]
@danilovaz tem alguem trabalhando nesta issue?

Existe algum tipo de orientação para fazer as traduções? Gostaria de ajudar.

**Comment by danilovaz** [comment]
@yanmagale Não, não tem ninguém nessa issue. Quer pegar essa? Se sim, me avisa aqui que já passo ela pra você.

Existe sim, dá uma olhadinha aqui: https://github.com/ember-brasil/website/blob/master/CONTRIBUTING.md

Qualquer coisa me chama no Telegram do Ember ou no Slack do Ember.

**Comment by yanmagale** [comment]
Quero pegar a issue sim @danilovaz . Vou trabalhar nela nesta semana :smile:

**Comment by danilovaz** [comment]
@yanmagale Show, colocamos essa issue pra você então. Obrigado pela ajuda!

**Comment by yanmagale** [comment]
@danilovaz eu que agradeço, fico feliz em ajudar.

Devo finalizar a tradução nesta semana, e ai libero o item pra revisão.

**Comment by aureliosaraiva** [comment]
Legal @yanmagale obrigado!
